tests/integration/property-check/src/test/java/org/glassfish/jersey/internal/config/SystemPropertiesConfigurationModelTest.java

```diff
@@ -77,6 +78,7 @@ public void allPropertyClassLoaded() throws IOException {
     }
 
     @Test
+    @Ignore("System.setSecurityManager is deprecated in JDK 19 and there is no replacement")
```

Comment:
good, but this can be ignored in a profile (pom.xml) designed solely for JDK 19 (while still working for other JDKs).

Reply:
Yep, this should have been excluded just for JDK 19+

Reply:
Right, fixed